### DISCUSSION TITLE
Objetivos de batalla automáticos

### DISCUSSION THREAD

**Discussion by Jeikonline** [Lurkars/gloomhavensecretariat#623] [Ideas]
Hola la aplicación está estupenda, solo que mis amigos y yo queremos tener un registro del orden, tipo y cantidad de monstruos que mata cada uno de los jugadores para así poder realizar automáticamente los objetivos de batalla, así cómo abrir puertas, saqueó etc

También crear una sección de las tarjetas de eventos como de ciudad y de camino

**Comment by Lurkars** [comment]
translated via deepl:

> Hi the application is great, only that my friends and I want to have a record of the order, type and quantity of monsters killed by each of the players so we can automatically perform the battle objectives, as well as how to open doors, looted etc.
> 
> Also create a section for event cards like city and road cards.

You can enable "Scenario Statistics" in settings which tracks damage dealt and monsters killed for every scenario.
Haven't thought about battle goal automation yet, but also thinks it shouldn't be too difficult to remember if goal is achieved.

Events are on my top list to support, but is just so much work, that it still will take month to come.
